teachopencadd/talktorials/T016_protein_ligand_interactions/talktorial.ipynb

```diff
@@ -4,58 +4,83 @@
    "cell_type": "markdown",
```

Comment:
<blockquote>PLIP only accepts PDB files, so we can pass the PDB file </blockquote><p>'only' here sounds like this is unfavorable for our example, maybe rephrase</p><p> </p><blockquote> processes each detected binding site</blockquote><p>How are binding sites 'detected'? I think PLIP uses all ligands (HETATM ids) in the PDB File and all residues within xA from them (or so)? Could you double check and add this info?</p><p> </p><p>[coming back to a comment in the theory, maybe the bullet points could be the same there, with short descriptions behind the colon]</p><p> </p><p>Not sure it is full clear here what the report contains: one entry per interaction, or per residue? Maybe an example helps?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>